octavia-cli/unit_tests/header_configuration.yaml

```diff
@@ -0,0 +1,3 @@
+#headers:
+#  - name: Authorization
+#    value: ${USER}
```

Comment:
Please add new line at the end of the file

Reply:
What do you think about renaming this file to `api_header_configuration.yaml` and add a comment on top of it to explain why/how it is used?

Reply:
@Imbruced don't you want to make this suggested change?

Reply:
Somehow forget to push that file.